pkg/controllers/management/node/controller.go

```diff
@@ -76,13 +81,17 @@ func Register(ctx context.Context, management *config.ManagementContext) {
 		clusterLister:             management.Management.Clusters("").Controller().Lister(),
 		schemaLister:              management.Management.DynamicSchemas("").Controller().Lister(),
 		credLister:                management.Core.Secrets("").Controller().Lister(),
+		userManager:               management.UserManager,
+		clusterManager:            clusterManager,
+		nodeInterface:             management.Core.Nodes(""),
```

Comment:
I don't think nodeInterface is ever use, can delete if that's the case.

Reply:
good eye